spec/tapioca/compilers/symbol_table_compiler_spec.rb

```diff
@@ -2281,6 +2282,8 @@ def inherited(s); end
         class Baz
           abstract!
 
+          # Don't need to silence warnings via without_ruby_warnings when calling
```

Comment:
This one is interesting. `pry-doc` will actually include documentation for methods created dynamically using `define_method`. This is coming from Sorbet.